manifests/n/Nextiva/NextivaONE/1.8.3/Nextiva.NextivaONE.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Automatically updated by the winget bot at 2022/Dec/17
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Nextiva.NextivaONE
+PackageVersion: 1.8.3
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
From the validation error, the installer is x64. 
![image](https://user-images.githubusercontent.com/114952565/208488320-09cb69e8-17ef-4774-8944-8ba2e1235cae.png)

We've recently had an issue where architectures were being omitted, so this might be another instance of that.